seq2science/rules/get_genome.smk

```diff
@@ -51,12 +51,20 @@ rule get_genome:
             genomepy install --genomes_dir {params.dir} {wildcards.assembly} NCBI    --annotation >> {log} 2>&1
         fi
         
-        # unzip annotation if downloaded, warn if required but empty
-        if [ -f {params.gtf} ]; then
-            gunzip {params.gtf} >> {log} 2>&1
-        elif echo {output} | grep -q annotation.gtf; then
-            echo '\nAnnotation for {wildcards.assembly} contains no genes. Select a different assembly or provide an annotation file manually.\n\n' > {log}
-            exit 1
+        # unzip annotation if downloaded and gzipped
+        if [ -f {params.gtf}.gz ]; then
+            gunzip -f {params.gtf}.gz >> {log} 2>&1
+        fi
+        
+        # if assembly has no annotation, or annotation has no genes, throw an warning
+        if [ ! -f {params.gtf} ] || $(grep -q "No genes found" {log}); then
+            echo '\nEmpty or no annotation found for {wildcards.assembly}.\n' | tee -a {log}
```

Comment:
@siebrenf why not redirect to log? 
`>> {log}`

Reply:
Because if you get this error at the start of your pipeline, its not too late to pick a different provider. So I think it might be nice to actually get the error, as well as getting it in the log.

Reply:
ok :+1: